cli/sub_command.go

```diff
@@ -761,6 +761,15 @@ func (c *subCmd) dumpMsg(msg *nats.Msg, stdout bool, filepath string, ctr uint)
 	serMsg.Data = msg.Data
 	serMsg.Reply = msg.Reply
 
+	if c.translate != "" {
+		data, err := filterDataThroughCmd(msg.Data, c.translate, "", "")
+		if err != nil {
+			fmt.Printf("%q\nError while translating msg body: %s\n\n", data, err.Error())
```

Comment:
below we use log.Printf() for logging errors, should probably do same here

also needs rebase